charts/trino/templates/_helpers.tpl

```diff
@@ -125,17 +125,33 @@ Code is inspired from bitnami/common
 {{- end -}}
 
 {{/*
-Create the name of the file auth secret to use
+Create the name of the file password auth secret to use
```

Comment:
What's a `file password auth`?

Reply:
Because later in _helpers.tpl there is "file group authentication" and before this PR there was a single secret/file for both the password and the group file. 
So I added it to the split.

Reply:
I figured you want to differentiate two names, but this whole sentence should be in correct English. What does `file password auth` mean in this context?

Reply:
Hmm, it means something like "file with passwords auth" to use in trino config by:
```
password-authenticator.name=file
file.password-file=
```

maybe phrases like : "Create the secret name for the file-based password authentication." will be better here ?

Reply:
How about:
> Create the secret name for the file-based authentication's password file.

Reply:
https://github.com/trinodb/charts/pull/231/commits/e007e824ebbdcc2103329eb5acbfaafc6e94b098